core/conn/odbc/src/odbc/Common/Global.h

```diff
@@ -935,7 +941,10 @@ typedef struct _SRVR_GLOBAL_Def
 		bzero(m_ProcName,sizeof(m_ProcName));
 		m_bNewConnection = false;
 		m_bNewService = false;
-
+        bzero(clientKeepaliveStatus, sizeof(clientKeepaliveStatus));
```

Comment:
Minor point: If we could keep the indentation consistent, that will make the code easier to read in the future. (Perhaps the new code uses tabs and the old code doesn't? Or maybe the old code uses tabs but the new code doesn't? In the past, we've tried to avoid tabs because they lead to these kinds of problems but we have not been successful in keeping them out.)

Reply:
That because old code uses tab which equals 8 blanks but i use 4 blanks directly（so the code aligned on my vim since I set one tab equals 4 blanks.） I will modify it to keep the indentation consistent, on github.